### ISSUE TITLE
Support unix domain sockets

### ISSUE THREAD

**Issue by rybakit** [basis-company/tarantool-admin#33]
Right now it's not possible to create a new unix socket connection via UI.

**Comment by nekufa** [comment]
Hello, @rybakit, looks like, it's not hard to fix this.
Can you share docker-compose file that can be used to establish unix socket connection?

**Comment by rybakit** [comment]
Try this (not tested):

*docker-compose.yml*

```yaml
version: '3.1'

services:
  tarantool:
    container_name: tarantool
    image: tarantool/tarantool:2
    restart: always
    volumes:
      - ./test.lua:/opt/tarantool/test.lua
    command: tarantool /opt/tarantool/test.lua
    ports:
      - "3301:3301"
  tarantool_admin:
    container_name: tarantool_admin
    image: quay.io/basis-company/tarantool-admin
    restart: always
    ports:
      - "8001:80"
    depends_on:
      - tarantool
    environment:
      - TARANTOOL_TCP_NODELAY=1
      - TARANTOOL_DATABASE_QUERY=1
      - TARANTOOL_CONNECTIONS='unix:///tmp/test.sock'
```

*test.lua*
```lua
#!/usr/bin/env tarantool

box.cfg {
    listen = '/tmp/test.sock',
    log_level = 5
}

-- allow group users to access the socket
require('fio').chmod(box.cfg.listen, tonumber('0664', 8))
```

**Comment by nekufa** [comment]
fixed in 0.3.15, use latest docker build.
